README.md

```diff
@@ -1,38 +1,130 @@
-trying to solve the same challenge as us
-and they are _not_ open-sourcing their code
-and/or guarding the user-feedback,
-they are _stuck_ in a _fixed_ mindset.
-We believe that _radical_ transparency and openness
-both in _what_ we do and _how_ we do it
-is _essential_ for succeeding in our _mission_.
-
-Only time will tell if making our product roadmap
-Public on GitHub is a good or bad idea, we think it's
-worth the _experiment_ just to discover the outcome.
-
-## _What_?
+on our _collective **goal**_.
+
+> **Note**: this document is a
+["work-in-progress"](https://en.wiktionary.org/wiki/work_in_progress#Noun),
+it's _definitely_ not "finished". <br />
+A _good_ product roadmap evolves over time
+in response to _feedback_;
+by definition it's _never_ "complete". <br />
+What you are reading is a _snapshot_
+of the roadmap as it is _right now_.
+If you ⭐ (bookmark) the repo 
+and return to it in future, it will _definitely_ be different
+because we update it fairly regularly.
+We use a Git repo so that we (_and anyone `else` in the community_)
+can _track_ the changes/evolution as we 
+[Learn, Build and Measure](http://theleanstartup.com/principles). <br />
+We welcome any/all feedback on it 
+_especially_ from people who are `new`
+to the 
+[`@dwyl`](https://github.com/dwyl/start-here#who) 
+community. <br />
+Please 
+[open an issue](https://github.com/dwyl/product-roadmap/issues)
+if you have any ideas/questions.
+Thanks! ✨ <br />
+
+
+<br />
+
+## _Why_ Are We Building An _App_?
+
+If there was an _existing_ 
+(Open Source Cross-platform Real-time Collaborative) 
+system 
+that allowed us to achieve _everything_ we have in mind for the dwyl App,
+we would _use_ it without hesitation.
+Sadly, we have not found any platform 
+that has a
+[_systematic_](https://en.wikipedia.org/wiki/System), 
+[goal oriented](https://en.wikipedia.org/wiki/Goal_orientation)
+and 
+[collaborative](https://en.wikipedia.org/wiki/Collaboration)
+way to _effectively_ let individuals and/or teams:
+
+1. **Set** **goals** that are ["SMART"](https://en.wikipedia.org/wiki/SMART_criteria)
+either freeform or using the [OKR](https://en.wikipedia.org/wiki/OKR) framework.
+
+2. **Capture** everything on one's mind 
+as (plaintext) `items`, (voice) `recordings` or `images`
+to declutter thoughts 
+and think more clearly/creatively
+
+3. **Categorise** the `items` into logical categories using `tags`
+(_ideally linked to specific goals_).
+
+4. **Transform** `items` of text into _actionable_ `items` 
+such as: 
+"Task" (_with context, deadline, progress, dependencies, etc._), 
+"Reminder", 
+"Calendar" (_Meeting, Date, Event, Travel, etc._), 
+"Ingredient" (Shopping list `item`), etc. 
+We want _complete_ versatility in the `tags` that can be applied to items
+so that our App can be used for _any_ purpose that helps people organise their lives.
+
+5. ***Track*** tasks and goal **progress _visually_**.
+
+6. ***Communicate*** with others what task we are focussing on
```

Comment:
```suggestion
6. **Communicate** with others what task we are focussing on
```

Reply:
Same here: "Communicate" is an _action_ word. So italics emphasise it. But I _agree_ that I use italics _way_ too often ... 🙄